docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
This is an important change, because it is necessary to do this to ensure efficient and scalable OCSP stapling support. Without this, Servers lack a lot of the controls needed to be able to enable it (since a client could force them to request a response with nonces).

It also resolves a deficiency from 6960, which the nextUpdate is optional. When it's optional, it means there is always a fresh response available - but it means the client cannot effectively cache the result, nor can servers staple it. This is unquestionably bad for security, performance, and privacy. It's also not consistent with [Microsoft's requirements](https://social.technet.microsoft.com/wiki/contents/articles/31633.microsoft-trusted-root-program-requirements.aspx#C_Revocation_Requirements), so it's unlikely any Member CA is relying on this.